savify/savify.py

```diff
@@ -239,7 +241,10 @@ def _download(self, track: Track) -> dict:
             **self.ydl_options,
         }
 
-        output_temp = output_temp.replace('%(ext)s', self.download_format)
+        output_temp = output_temp.replace(
+            '%(ext)s',
+            download_extension_map.get(self.download_format, self.download_format),
```

Comment:
There's an error line 246. It should be:

  output_temp = output_temp.replace('%(ext)s', format_extension_map.get(self.download_format, self.download_format),)

Reply:
youre right, idk where this comes from - fixed